test/Bin/GameTest/GameTests_1500.cpp

```diff
@@ -26,6 +26,20 @@ GAME_TEST(Issues, Issue1510) {
     EXPECT_LE(actorDistTape.max(), meleeRange);
 }
 
+GAME_TEST(Issues, Issue1515) {
+    // No dispel magic sound
+    auto soundsTape = tapes.sounds();
+    test.playTraceFromTestData("issue_1515.mm7", "issue_1515.json");
+    EXPECT_TRUE(soundsTape.flattened().contains(SOUND_RechargeItem)); // dispel magic
+}
+
+GAME_TEST(Issues, Issue1515_1524) {
```

Comment:
`GAME_TEST(Issues, Issue1524)`?

Reply:
The tests are failing though X)

Reply:
Yeah spotted that one.
I need to slightly mod how result from playing the sound is tested. At the moment it fails because of no sound device, whereas the tests need to check whether we tried to queue the sound or not.